lib/Notification/styles.scss

```diff
@@ -1,9 +1,7 @@
 .notification {
-	font-size: 13px;
-	line-height: 1.75em;
-	padding: 10px 15px;
+	font-size: $typo-size-slight;
+	padding: $layout-spacing-base/2 $layout-spacing-base*0.75;
```

Comment:
Would it make sense to add fixed values for these, or to contain them in separate values without making loads of math already? :) (just pinging some thoughts, really!)